e2e/cypress/integration/collapsed_reply_threads/crt_settings_spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+//
+//  Group: @collapsed_reply_threads
+
+describe('CollapsedReplyThreads', () => {
```

Comment:
nit, same suggestion to others.
```suggestion

//  Group: @collapsed_reply_threads

describe('Collapsed Reply Threads', () => {
```